### DISCUSSION TITLE
Change Browser title

### DISCUSSION THREAD

**Discussion by immohacker** [Budibase/budibase#10942] [Help] [answered]
Hi 
i try to change the browser title.
i would change it to the Customer Name so i can see which Customer is opened, because i open the customer details in an new tab.
So if i opened more customers also more tabs, so in the title i can see wich customer it is.

i tried some code variants in an embed component, but nothing works.

my tries
first try
`<script>
    document.title = "Neuer Titel";
</script>
`
second try

`<script>
window.onload = function(){
    document.title = "Neuer Titel";
}
</script>
`
third try
with timeout

`<script>
    setTimeout(function(){
        document.title = "Neuer Titel";
    }, 1000); // Waits 1000 Milliseconds(1 Second) before execute code
</script>
`
Nothing works.
Have anybody an idea?
Thank you!
Best regards Bonkersdeluxe

**Comment by ConorWebb96** [comment, marked as answer]
Hey @immohacker,

You can do this, however what I'm going to show is a hack. By default I believe that Budibase remove scripts tags from the embed components. The below code should work for you inside an embed component.

```
<img style="display: none;" src onerror="document.title = 'Neuer Titel'" >
```
<img width="771" alt="Screenshot 2023-06-19 at 11 23 29" src="https://github.com/Budibase/budibase/assets/126772285/e8c9d7da-a301-4db2-957d-61416636f754">

I hope this helps!

**Comment by immohacker** [comment]
Hi ConorWebb96
Thank you!
It Works!
Best regards Bonkersdeluxe
